nifi-nar-bundles/nifi-extension-utils/nifi-listed-entity/src/main/java/org/apache/nifi/processor/util/list/AbstractListProcessor.java

```diff
@@ -761,6 +763,7 @@ public void listByTrackingTimestamps(final ProcessContext context, final Process
         }
 
         if (entityList == null || entityList.isEmpty()) {
+            getLogger().debug(String.format("There is no data to list with the criteria minTimestampToListMillis=%d. Yielding.", minTimestampToListMillis));
```

Comment:
slf4j's api provides parameterized logging which can be used here instead of String::format.

```suggestion
            getLogger().debug("There is no data to list with the criteria minTimestampToListMillis={}. Yielding.", minTimestampToListMillis);
```